eZ/Publish/Core/MVC/Symfony/Security/User/BaseProvider.php

```diff
@@ -68,7 +68,7 @@ public function refreshUser(CoreUserInterface $user)
      */
     public function supportsClass($class)
     {
-        $supportedClass = 'eZ\\Publish\\Core\\MVC\\Symfony\\Security\\User';
+        $supportedClass = 'eZ\\Publish\\Core\\MVC\\Symfony\\Security\\UserInterface';
```

Comment:
This method could be simplified to 

```
return $class === UserInterface::class || is_subclass_of($class, UserInterface::class);
```

Reply:
Just following what was already there.  I have made thi chnge